packages/by-name/cloud-api-adaptor/package.nix

```diff
@@ -3,13 +3,16 @@
 
 {
   lib,
+  pkgs,
```

Comment:
We don't really pass the top-level `pkgs` to functions in our tree. (idk why, tbh)
You'd probably want to pass `iptables` only instead.

Reply:
Done